app/views/waste_exemptions_engine/exemptions_forms/new.html.erb

```diff
@@ -30,7 +30,7 @@
               <td>
                 <div class="multiple-choice">
                   <%= f.check_box :exemptions, { :multiple => true, id: "exemptions_form_checkbox-#{exemption.code}", checked: @exemptions_form.matched_exemption_ids.include?(exemption.id) }, exemption.id, nil %>
-                  <%= f.label "checkbox-#{exemption.code}", exemption.code %>
+                  <label for="exemptions_form_checkbox-<%= exemption.code %>"><%= exemption.code %><span class="visually-hidden"> <%= exemption.summary %></span></label>
```

Comment:
It's possible to do this using the `f.label` block syntax: https://github.com/DEFRA/waste-exemptions-engine/blob/8266da8ede9c939822f0b49b4466c4b82d1b56ee/app/views/waste_exemptions_engine/shared/_manual_address.html.erb#L11-L14

Think we should do that over raw HTML for consistency's sake.

Reply:
Ooh, awesome. Just my lack of knowledge there. Will update.